assemblies/collaborating-on-workbenches-in-code-server-by-using-git.adoc

```diff
@@ -0,0 +1,21 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+[id="collaborating-on-workbenches-in-code-server-by-using-git_{context}"]
+= Collaborating on workbenches in code-server by using Git
+
+[role='_abstract']
+If your notebooks or other files are stored in Git version control, you can import them from a Git repository onto your notebook server to work with them in code-server. When you are ready, you can push your changes back to the Git repository so that others can review or use your models.
```

Comment:
> onto your notebook server to work with them in code-server

```suggestion
If your notebooks or other files are stored in Git version control, you can clone a Git repository to work with them in code-server. When you are ready, you can push your changes back to the Git repository so that others can review or use your models.
```

I imagine that user won't see any difference between a "notebook server" and "code-server". As a user, I am checking out the repository into (my instance of) code-server, no? And I see nothing that could be called notebook server anywhere in the process.